### PR TITLE
if no offline editor is available, use OO/Cool/Text

### LINKED ISSUE #5340: Synchronized files can not be edited in Onlyoffice
### Steps to reproduce
1. Synchronize a folder containing a document/spreadsheet/...
2. Try to edit it in Onlyoffice
3. 

### Expected behaviour
Onlyoffice should open the file

### Actual behaviour
Screenshot_20200128_125452

### Environment data
Android version: 9

Device model: Huawei P20

Stock or customized system: Stock

Nextcloud app version: 3.10.0

Nextcloud server version: 18.0.0

Onlyoffice app: 4.1.4

see #5329 

If a file is synchronized locally it won't open in Onlyoffice. (see screenshot).

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix #5340 

- download an OO capable file
- have no local app that can open it
- previously: "no app to handle it"
- now: uses OO/Cool/Text if available, else "no app to handle it"

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### Testing 
Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

[unit tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests)
[instrumented tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests)
[UI tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests)

- [x] Tests written, or not not needed

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java [hunk 1]
```diff
@@ -278,16 +280,25 @@ public void openFile(OCFile file) {
...
+                    Account account = fileActivity.getAccount();
+                    OCCapability capability = fileActivity.getStorageManager().getCapability(account.name);
+                    if (capability.getRichDocumentsMimeTypeList().contains(file.getMimeType()) &&
+                        android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP &&
+                        capability.getRichDocumentsDirectEditing().isTrue()) {
+                        openFileAsRichDocument(file, fileActivity);
+                        return;
+                    } else {
+                        DisplayUtils.showSnackMessage(fileActivity, R.string.file_list_no_app_for_file_type);
+                        return;
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java]: the `return` calls aren't needed

Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java]: They are, otherwise the next block starting at Line 305 will be executed.

Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java]: 👍

**Pull request merged**